### PR TITLE
changelog update for v9

### PR DESCRIPTION
**Pull request by JoeChapman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # HOF ChangeLog
 
+## 2016-06-08, Version 9.0.0 (Unstable), @joechapman
```
Comment (easternbloc) [inline CHANGELOG.md:3]: `(Unstable)` ->

Comment (easternbloc) [inline CHANGELOG.md:3]: If this release is not stable we should probably unpublish that version...

**Comment by JoeChapman** [comment]
I done did and went and published a verion that included the latest hof-controllers but did not export the latest addition (start-controller) so I don't want people to use this version

**Comment by easternbloc** [comment]
> I done did and went and published a verion that included the latest hof-controllers but did not export the latest addition (start-controller) so I don't want people to use this version

`npm unpublish hof@9.0.0`

**Comment by easternbloc** [comment]
I have unpublished that version from npm. We should wait till we're ready to make the next release.

**Pull request closed without merging**